src/main/scala/mesosphere/marathon/core/election/ElectionService.scala

```diff
@@ -0,0 +1,30 @@
+package mesosphere.marathon.core.election
+
+import scala.concurrent.Future
+
+trait LeadershipAbdication {
```

Comment:
I like small traits, but perhaps this method could just move into ElectionService?